locksmith/config/config.js

```diff
@@ -1,4 +1,11 @@
-require('dotenv').config()
+const path = require('path')
+const dotenv = require('dotenv')
+
+const unlockEnv = process.env.UNLOCK_ENV || 'dev'
+
+dotenv.config({
```

Comment:
Is this an optional lookup? If this file is not available, will it look for a standard `.env` file and subsequently the environment variable of the running context?

Reply:
No, you need to put its content inside of `.env.dev.local`.